dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.webBundle.internal.ee-9.0.feature

```diff
@@ -4,10 +4,18 @@ symbolicName = io.openliberty.webBundle.internal.ee-9.0
 singleton=true
 visibility = private
 
--features= com.ibm.websphere.appserver.servlet-5.0; ibm.tolerates:="6.0"
+-features= io.openliberty.servlet.internal-5.0
```

Comment:
Did you intend to remove tolerates for 6.0 here?

Reply:
Yes because there is a ee-10.0 version of the feature that does not expose the SPI.